core/src/main/java/nl/nn/adapterframework/jdbc/JdbcFacade.java

```diff
@@ -259,6 +260,16 @@ public void iterateOverStatistics(StatisticsKeeperIterationHandler hski, Object
 		hski.handleStatisticsKeeper(data, connectionStatistics);
 	}
 
+	@Override
+	@Deprecated
+	@ConfigurationWarning("Please use attribute dataSourceName instead")
+	public void setJmsRealm(String jmsRealmName) {
+		super.setJmsRealm(jmsRealmName);
+		if(StringUtils.isEmpty(getJmsRealmName())) { //Validate that the realm was found
+			throw new IllegalStateException("JmsRealm ["+jmsRealmName+"] not found");
```

Comment:
Het is niet 'not found', maar 'not specified'. En dat is vervolgens eigenlijk geen probleem

Reply:
Nee, jouw aanname klopt niet...